pkg/pb/translate/v1/translate.pb.go

```diff
@@ -1,6 +1,6 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.32.0
+// 	protoc-gen-go v1.33.0
```

Comment:
@VladislavsPerkanuks it is a generated file, you know. :)

Reply:
it was changed by `earthly +proto`

Reply:
It is possible, do you want me to change to specific version ?

Reply:
okay, done it, nevertheless :)

Reply:
Perfect.